frontend/packages/kubevirt-plugin/src/components/constants.js

```diff
@@ -0,0 +1 @@
+export const DASHES = '---';
```

Comment:
If it makes sense, we can reuse the `DASH` constant from `console-shared` package.

Reply:
from #1682, under review & change